datasets/code_x_glue_cc_clone_detection_big_clone_bench/README.md

```diff
@@ -0,0 +1,152 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- C++
+licenses:
+- other-C-UDA
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets: []
+task_categories:
+- text-classification
+task_ids:
+- semantic-similarity-classification
+---
+# Dataset Card for "code_x_glue_cc_clone_detection_big_clone_bench"
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+
+
+## [Dataset Description](#dataset-description)
+
+ 
+- **Homepage:** https://github.com/microsoft/CodeXGLUE/tree/main/Code-Code/Clone-detection-BigCloneBench
+
+ 
```

Comment:
Can you remove all the empty spaces in the markdown ?
You can simply format the markdown to not have two consecutive empty lines in a row

Reply:
Yes, I will have a look at the source jinja template: the markdown was generated, and so to remove empty lines it will make the template itself a big difficult to read, so I may do a post process instead, removing consecutives empty lines.